libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -614,7 +635,6 @@ private async Task<DialogTurnResult> CheckForMultiTurnPromptAsync(WaterfallStepC
                 if (answer.Context != null && answer.Context.Prompts.Count() > 0)
```

Comment:
> if (answer.Context != null && answer.Context.Prompts.Count() > 0) [](start = 15, length = 66)

This 'if' can be inverted: if(!answer.Context != null || answer.Context.Prompts.Count() == 0) return;

Reply:
Discussed.

---
In reply to: [433510646](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r433510646) [](ancestors = 433510646)